css/style.css

```diff
@@ -1,4 +1,572 @@
+  color: #000;
+  text-align: center;
+}
+
+.banner__text {
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 18px;
+  text-align: center;
+}
+
+.banner__img_background {
+  width: 1514px;
+  margin: -10px auto 0;
+  display: block;
+  position: relative;
+  z-index: 0;
+}
+
+.banner__img_red {
+  display: block;
+  height: 450px;
+  margin: -220px auto 0;
+  position: relative;
+  z-index: 999;
+}
+
+.test .test__row {
+  width: 802px;
+  margin: 27px auto 0;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: justify;
+  -ms-flex-pack: justify;
+  justify-content: space-between;
+}
+
+.test__row_right {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  width: 378px;
+  -webkit-box-pack: justify;
+  -ms-flex-pack: justify;
+  justify-content: space-between;
+}
+
+.test__row_left {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  width: 320px;
+  -webkit-box-pack: justify;
+  -ms-flex-pack: justify;
+  justify-content: space-between;
+}
+
+.test__column:nth-child(3) {
+  margin: 10px 0 0 0;
+}
+
+.test__item {
+  padding: 0 7px 0 0;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+}
+
+.test__right {
+  padding: 0;
+}
+
+.test__column_testdrive {
+  padding: 10px 0 0 0;
+}
+
+.test__column_testdrive.test__img {
+  margin: 2px 2px 0 0;
+}
+
+.test__column_testdrive.link_test__item {
+  text-decoration: none;
+  color: #000;
+}
+
+.test__body {
+  font-weight: 600;
+  font-size: 35px;
+  margin: 0;
+}
+
+.test__body_small {
+  font-size: 20px;
+}
+
+.test__body_bottom {
+  font-size: 12px;
+  line-height: 15px;
+  margin: 1px 0 0 0;
+}
+
+.test__body_testdrive {
+  font-weight: bold;
+  font-size: 13px;
+  line-height: 16px;
+  text-align: center;
+  text-transform: uppercase;
+  padding: 7px 0 0 7px;
+}
+
+.test__body_testdrive:hover {
+  color: #e51937;
+  -webkit-transition: ease-in-out .5s;
+  -o-transition: ease-in-out .5s;
+  transition: ease-in-out .5s;
+  }
+
+.btn {
+  width: 160px;
+  height: 40px;
+  border: 2px solid #e51937;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+  font-weight: bold;
+  font-size: 13px;
+  text-align: center;
+  text-transform: uppercase;
+  color: #e51937;
+  padding: 10px 0 0 0;
+}
+
+.btn__test {
+  color: #e51937;
+  border: 2px solid #e51937;
+}
+
+.btn__test:hover {
+  background-color: #e51937;
+  color: #fff;
+  -webkit-transition: ease-in-out .5s;
+  -o-transition: ease-in-out .5s;
+  transition: ease-in-out .5s;
+}
+
+.button {
+  margin: 5px 0 0 0;
+}
+
+.test__img {
+  display: block;
+  margin: 5px  auto 0;
+  width: 100%;
+  height: 100%;
+  -ms-flex-item-align: start;
+  align-self: flex-start;
+}
+
+.test__arrow {
+  margin: 39px auto 0;
+  width: 30px;
+}
+
+@media (max-width: 1100px) {
+  .banner__img_background {
+    width: 1000px;
+    height: 300px;
+  } 
+}
+
+@media (max-width: 768px) {
+  .banner__img_background {
+    width: 700px;
+    height: 270px; 
+  }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected newline before "}" of a multi-line block (block-closing-brace-newline-before)](https://app.codacy.com/gh/WebHeroSchool/chendler_tesla/pullRequest?prid=5700328)